src/__tests__/unit/evaluation-options.test.ts

```diff
@@ -1,5 +1,6 @@
 import { EvaluationOptionsEvaluator } from '../../contract/EvaluationOptionsEvaluator';
 import { WarpFactory } from '../../core/WarpFactory';
+import { SourceType } from '../../core/modules/impl/WarpGatewayInteractionsLoader';
```

Comment:
I would add some dumb intergration test, that would deploy a contract with a manifest set to sourceType: sequencer, and then create some transactions (which in case of local testing are always considered as 'direct' Arweave transactions) - and verify that the SDK always does not load anything.

Reply:
As discussed this is not that easy. It will probably require dedicated config for local env and some fetch mocking. And after all that it may not be that reliable.